### PR TITLE
Get user's name and add to text on top bar when user is logged in

### PR DESCRIPTION
**Pull request by PriyaVarra**
This PR makes a minor update by fetching the name the user signed up with from Firebase and adding it to the text on the top bar so that it says "Hi, {name}! Planning a road trip? Enter ..." when they are logged in. When they logout, the text goes back to normal.

### CODE REVIEW

File: where-to-travel/src/main/webapp/script.js [hunk 1]
```diff
@@ -808,19 +814,28 @@ function getLoginHtml() {
...
  */
-function getUserDashHtml(user) {
-  return `<img onclick="showModal(${INFO_HTML_PATH})" class="btn btn-icon" src="icons/help.svg">
-          Display Saved:
-          <label class="switch btn">
-            <input type="checkbox">
-            <span class="slider round"></span>
-          </label>
-          <a class="btn btn-outline-primary btn-color" style="color: #049688;" id="logout">Logout</a>`;
+function getUserDashHtml() {
```
Comment (andy7809) [inline where-to-travel/src/main/webapp/script.js:823]: This is a much better way to do this function. Why pass around a User object if we have the info we need in firebase?
Good job

**Review by andy7809 — APPROVED** [review]
LGTM

**Review by emmapaczkowski — APPROVED** [review]
LGTM

File: where-to-travel/src/main/webapp/index.html [hunk 2]
```diff
@@ -19,10 +19,10 @@
       <header class="bmd-layout-header">
         <div class="navbar bg-faded">
           <div class="text-center" style="width:330px">
-            <h1 class="navbar-brand text-center text-color font-size-large"> <img src="icons/pin.svg" width="20" height="38"> Where to?</h1>
+            <h1 class="navbar-brand text-center text-color font-size-large"> <img src="icons/pin.svg" width="20" height="38"> Where to? </h1>
```
Comment (saileshsimhadri) [inline where-to-travel/src/main/webapp/index.html:22]: Did you mean to add this space?

**Review by saileshsimhadri — COMMENTED** [review]
LGTM

File: where-to-travel/src/main/webapp/script.js [hunk 3]
```diff
@@ -808,19 +814,28 @@ function getLoginHtml() {
...
+  return new Promise(function(resolve) { 
+     firebase.database().ref('users/'+ firebase.auth().currentUser.uid)
+      .once('value', function(userSnapshot){
+        const name = userSnapshot.val().name;
+        document.getElementById('top-text').innerHTML =
+          'Hi, ' + name + '! ' + TOP_INFO_STR;
```
Comment (wjiamin) [inline where-to-travel/src/main/webapp/script.js]: Consider using string.replace for this instead? Have a constant similar to 

'Hi, $username!'

File: where-to-travel/src/main/webapp/script.js [hunk 4]
```diff
@@ -808,19 +814,28 @@ function getLoginHtml() {
...
-            <input type="checkbox">
-            <span class="slider round"></span>
-          </label>
-          <a class="btn btn-outline-primary btn-color" style="color: #049688;" id="logout">Logout</a>`;
+function getUserDashHtml() {
+  return new Promise(function(resolve) { 
+     firebase.database().ref('users/'+ firebase.auth().currentUser.uid)
+      .once('value', function(userSnapshot){
+        const name = userSnapshot.val().name;
+        document.getElementById('top-text').innerHTML =
```
Comment (wjiamin) [inline where-to-travel/src/main/webapp/script.js]: Do we want to create a constant for 'top-text'?

**Review by wjiamin — COMMENTED** [review]
LGTM

File: where-to-travel/src/main/webapp/index.html (hunk 2 above)
Comment (PriyaVarra) [reply where-to-travel/src/main/webapp/index.html:22]: Yes, I did it to be consistent with the leading space.

File: where-to-travel/src/main/webapp/script.js (hunk 4 above)
Comment (PriyaVarra) [reply where-to-travel/src/main/webapp/script.js]: Fixed!

File: where-to-travel/src/main/webapp/script.js (hunk 3 above)
Comment (PriyaVarra) [reply where-to-travel/src/main/webapp/script.js]: Fixed!

**Pull request merged**